tests/parametric/test_telemetry.py

```diff
@@ -89,6 +89,91 @@ def test_library_settings(self, library_env, test_agent, test_library):
             assert cfg_item.get("origin") == "default", "Unexpected origin for '{}'".format(apm_telemetry_name)
 
 
+@scenarios.parametric
+@rfc("https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o")
+@features.telemetry_app_started_event
```

Comment:
is it really this feature? 🤔

Reply:
@cbeauchesne I blindly copied that over from the Test_Environment class below 😄
Technically this behavior is expected when app has started... and all these env vars have been set. But this is also part of the tracing_configuration_consistency effort that we've been working on, and that @zacharycmontoya added a `feature` for in https://github.com/DataDog/system-tests/pull/2980. Should I mark this with `@features.tracing_configuration_consistency` instead, or does that *not* make sense because this is telemetry-specific? WDYT?

Reply:
The RFC filled for this feature is https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o/edit#heading=h.ixldy9ty2xy5

If the behavior your testing is described in this RFC, it's definitly the one to use. otherwise, you can either create a new feature, or still use `tracing_configuration_consistency`, up to you/your team to define how you want to define  this feature.